packages/react-instantsearch-dom-maps/src/GoogleMapsContext.ts

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+export type GoogleMapsContextProps = {
+  google: typeof google;
+  instance: google.maps.Map;
+}
+
+const GoogleMapsContext = React.createContext<Partial<GoogleMapsContextProps>>({});
```

Comment:
We should avoid the partial otherwise we'll have to use the `!` operator when we use the context. We can provide default value.